tools/get-change-log-for-release.js

```diff
@@ -11,9 +11,11 @@ const main = async () => {
     validateCommandLineArguments(params);
 
     const gitLogs = await getGitLogs(params.from, params.to);
-
-    const outputContent = generateOutputContent(gitLogs, params.to);
```

Comment:
Maybe rename this method to `generateCSVContent`, assuming it doesn't break API elsewhere.